fetch-csv/src/main.js

```diff
@@ -110,12 +110,12 @@ function findLineSeparator(content) {
 
 function fetchData(url) {
   if (!url || !url.match(/^https?:\/\/.+$/g)) {
-    sendUserError('Input error: Invalid URL');
+    sendUserError('"' + url + '" is not a valid url.');
```

Comment:
Backtick string here? 

    `"${url}" ...`

Reply:
Not supported in apps scripts, unfortunately.